ocaml/testsuite/tests/typing-layouts/annots_beta.ml

```diff
@@ -516,7 +516,7 @@ Error: This definition has type 'b -> 'b which is less general than
        The layout of 'a is value, because
          of the annotation on the universal variable 'a.
        But the layout of 'a must be a sublayout of immediate, because
-         of the annotation on the universal variable 'a.
+         of the definition of f_imm at line 1, characters 4-9.
```

Comment:
This line is definitely an improvement. Good. But the first line of this error (which GitHub won't allow a comment on) is still not great, without having `'b : immediate` somewhere nearby. Looking at the open tickets around error messages, I don't think there's one for this infelicity. Could you add one?

Reply:
Opened a ticket for this.